addons/common/stringtable.xml

```diff
@@ -10,5 +10,9 @@
             <French>Community Base Addons - Composants communs</French>
             <Polish>Community Base Addons - Ogólne Komponenty</Polish>
         </Key>
+        <Key ID="STR_cba_common_need_mission_start">
+            <English>You must first start a mission.</English>
```

Comment:
Dosn't it sound too personal? Maybe something like `Mission must be started.`?

Reply:
>An example of a verb used in the imperative mood is the English sentence "Please be quiet". Such imperatives imply a second-person subject (you)[.]

Using the imperative is the best way to guide the user by tooltip in my experience.